src/main/java/net/mojodna/osm2orc/standalone/model/Changeset.java

```diff
@@ -0,0 +1,115 @@
+package net.mojodna.osm2orc.standalone.model;
+
+
+import org.openstreetmap.osmosis.core.domain.common.TimestampContainer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Changeset {
+
+    public static final String ID = "id";
+    public static final String CREATED_AT = "created_at";
+    public static final String CLOSED_AT = "closed_at";
+    public static final String OPEN = "open";
+    public static final String NUM_CHANGES = "num_changes";
+    public static final String USER = "user";
+    public static final String UID = "uid";
+    public static final String MIN_LAT = "min_lat";
+    public static final String MAX_LAT = "max_lat";
+    public static final String MIN_LON = "min_lon";
+    public static final String MAX_LON = "max_lon";
+    public static final String COMMENTS_COUNT = "comments_count";
+    public static final String TAG = "tag";
+
+    private static long instanceCount = 0;
+
+    private long id;
+    private TimestampContainer createdAt;
+    private TimestampContainer closedAt;
+    private boolean open;
+    private long numChanges;
+    private String user;
+    private Long uid;
+    private Double minLat;
+    private Double maxLat;
+    private Double minLon;
+    private Double maxLon;
```

Comment:
Can you expose these as `String`s here so we don't lose precision going from `String` → `Double` → `BigDecimal`?